index.pug

```diff
@@ -209,32 +189,27 @@ section#testimonial
                     h5 Anubha Maneshwar
                     p Founder, Girlscript
                   li.testi-logo
-                    img.lazy(data-src="/assets/education/girl-script-logo.jpg"
-                    )
+                    img.lazy(data-src="/assets/education/girl-script-logo.jpg", alt="GirlScriptLogo")
                   li.quotes
-                    img.lazy(data-src="/assets/education/quotes.png"
-                    )
+                    img.lazy(data-src="/assets/education/quotes.png", alt="Quotes")
         .one-slide
           .testimonial.testi1.video
             .message
-             img.lazy(data-src="/assets/education/video-testi.png"
-              )
+             img.lazy(data-src="/assets/education/video-testi.png", alt="Videotestimonial")
               .testi-msg
                 ul
                   li
                     h5 Anubha Maneshwar
                     p Founder, Girlscript
                   li.testi-logo
-                    img.lazy(data-src="/assets/education/1.png"
-                    )
+                    img.lazy(data-src="/assets/education/1.png", alt="EducationPic1")
                   li.quotes
-                    img.lazy(data-src="/assets/education/quotes-w.png"
-                    )
+                    img.lazy(data-src="/assets/education/quotes-w.png", alt="quotesW")
       .controller
         .control.prev(style="top: 0px; right: 0px")
-          img(src="img.lazy(data-src="/assets/education/arrow.svg")
+          img(src="img.lazy(data-src="/assets/education/arrow.svg", alt="arrow")
```

Comment:
Still there is an error here! and 
here
https://github.com/dviotech/rp_testing/pull/1/commits/e7e1581ad936940861ff4dd0e5ae0b8fc5857a25#diff-30bd6a76381095ceb9f709ac2217a6609ab50998ed55e8191de0367f6baf5c04R212